fun_3000/word2vec.py

```diff
@@ -9,14 +9,16 @@
 logging.basicConfig(format='%(asctime)s: %(levelname)s : %(message)s', level=logging.INFO)
 
 class MySentences(object):
```

Comment:
use https://radimrehurek.com/gensim/models/word2vec.html#gensim.models.word2vec.LineSentence instead so we don't have to wrap our own